ruby-training/trainees/alan/test_katas/03/JadenCase/jaden_case.rb

```diff
@@ -0,0 +1,13 @@
+
+
+class JadenCase
+
+    def self.array_of_capitalize_words(sentence)
```

Comment:
Should be `array_of_capitalized_words` to be grammatically correct, but this code is a lot better in general!